src/axios/globalAxios.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import axios, { AxiosRequestConfig, AxiosResponse, AxiosError } from "axios";
+import { GET_CURRENCIES, LOG_IN } from "../config/config";
+import { getFromLocalStorage } from "../storageUtils";
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux'
+import type { RootState, AppDispatch } from '../redux/store'
+
+import { useAppDispatch } from "../redux/store";
+import { startLoading, stopLoading } from "../redux/slices/sliceLoader";
+
+const GlobalAxios: React.FC = () => {
```

Comment:
global axios should be just a ts file, no need to create component, just import the file

Reply:
yes, that how it was. but becuase I use the "useSelector" hook I had to call it from a component

Reply:
I mean I use the "useAppDispatch();" hook